test-guru/db/seeds.rb

```diff
@@ -12,21 +12,25 @@
                                 {name: 'JS'},
                                 {name: 'CSS'}])
 
-  tests = Test.create!([{ title: 'Ruby', level: 0, category_id: categories[0].id },
-                      { title: 'Ruby on Rails', level: 1, category_id: categories[0].id },
-                      { title: 'HTML', level: 0, category_id: categories[1].id },
-                      { title: 'HTML 5', level: 1, category_id: categories[1].id },
-                      { title: 'JS', level: 0, category_id: categories[2].id },
-                      { title: 'CSS', level: 0, category_id: categories[3].id }])
+  users = User.create!([{ name: 'john@example.com'},
+                      { name: 'bob@example.com'},
+                      { name: 'jane@example.com'}])
 
-  questions = Question.create!([{ text: 'Класс это...', answer: 'Объект'},
-                             { text: 'Объект это...', answer: 'Класс'}])
+  tests = Test.create!([{ title: 'Ruby', level: 0, author_id: users[1].id, category: categories[0] },
```

Comment:
```suggestion
  tests = Test.create!([{ title: 'Ruby', level: 0, author: users[1], category: categories[0] },
```
для категории поправил, а для автора?